packages/clangml-transforms/clangml-transforms.0.25/opam

```diff
@@ -0,0 +1,35 @@
+opam-version: "2.0"
+synopsis: "Code transformers for clangml"
+maintainer: ["Thierry Martinez <thierry.martinez@inria.fr>"]
+authors: ["Thierry Martinez <thierry.martinez@inria.fr>"]
+license: "BSD-2-Clause"
+homepage: "https://gitlab.inria.fr/memcad/clangml-transforms"
+doc: "https://gitlab.inria.fr/memcad/clangml-transforms"
+bug-reports: "https://gitlab.inria.fr/memcad/clangml-transforms/issues"
+depends: [
+  "ocaml" {>= "4.03.0" & < "4.12.0"}
+  "dune" {>= "2.2.0"}
```

Comment:
According to the dune-project file, you are using lang 1.11. This is stricter, so it is not a problem, but maybe you want to relax it.

Reply:
Fixed in https://github.com/ocaml/opam-repository/pull/16446/commits/c5f23b5623cd7e084a3e675870ccc187a74ac275 (and upstream: https://gitlab.inria.fr/memcad/clangml-transforms/-/commit/98d2e9b0aca914fca30f8ba0084f7a51e35fb87e )
Thanks!